### PR TITLE
Feature/related products

### PR DESCRIPTION
**Pull request by jaimelr**
I created a partial to insert in the product detail page which shows related products to the one being watched by the user.

This branch will merge to `feature/product-details` to then merge that on master (`MH-Team2`).

### CODE REVIEW

File: theme6/source/javascripts/related_products.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+const leftArrow = document.getElementById('leftArrow');
+const rightArrow = document.getElementById('rightArrow');
+
+leftArrow.addEventListener('click', () => {
+  changeImage(-1);
+});
+
+rightArrow.addEventListener('click', () => {
```
Comment (magmahound) [inline theme6/source/javascripts/related_products.js]: 'arrow function syntax (=>)' is only available in ES6 (use esnext option).

File: theme6/source/javascripts/related_products.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    i = 0;
+  }
+
+  document.getElementById('featuredToy').src = 'images/' + images[i] + '.png';
+}
+
+const leftArrow = document.getElementById('leftArrow');
+const rightArrow = document.getElementById('rightArrow');
+
+leftArrow.addEventListener('click', () => {
```
Comment (magmahound) [inline theme6/source/javascripts/related_products.js]: 'arrow function syntax (=>)' is only available in ES6 (use esnext option).

File: theme6/source/javascripts/related_products.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  }
+  else if (i > images.length - 1) {
+    i = 0;
+  }
+
+  document.getElementById('featuredToy').src = 'images/' + images[i] + '.png';
+}
+
+const leftArrow = document.getElementById('leftArrow');
+const rightArrow = document.getElementById('rightArrow');
```
Comment (magmahound) [inline theme6/source/javascripts/related_products.js]: 'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).

File: theme6/source/javascripts/related_products.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    i = images.length - 1;
+  }
+  else if (i > images.length - 1) {
+    i = 0;
+  }
+
+  document.getElementById('featuredToy').src = 'images/' + images[i] + '.png';
+}
+
+const leftArrow = document.getElementById('leftArrow');
```
Comment (magmahound) [inline theme6/source/javascripts/related_products.js]: 'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).

**Review by Ilyeo — APPROVED** [review]
LGTM

**Pull request closed without merging**